README.md

```diff
@@ -84,48 +149,54 @@ ts.execute(Policy.PERFORMANCE);
 
 Further details and instructions on how to enable this feature can be found here.
 
-* Dynamic reconfiguration: [https://dl.acm.org/doi/10.1145/3313808.3313819](https://dl.acm.org/doi/10.1145/3313808.3313819)
-
+* Dynamic
+  reconfiguration: [https://dl.acm.org/doi/10.1145/3313808.3313819](https://dl.acm.org/doi/10.1145/3313808.3313819)
 
 ## 5. How to Use it in your Projects?
 
-You can import the API and start using TornadoVM. Set this in the `pom.xml` file. 
+You can import the API and start using TornadoVM. Set this in the `pom.xml` file.
 
 ```xml
-   <repositories>
-     <repository>
-       <id>universityOfManchester-graal</id>
-       <url>https://raw.githubusercontent.com/beehive-lab/tornado/maven-tornadovm</url>
-     </repository>
-   </repositories>
-  
-   <dependencies>   
-      <dependency>
-         <groupId>tornado</groupId>
-         <artifactId>tornado-api</artifactId>
-         <version>0.8</version>
-      </dependency>
-      <dependency>
-         <groupId>tornado</groupId>
-         <artifactId>tornado-matrices</artifactId>
-         <version>0.8</version>
-      </dependency>
-   </dependencies>
+
+<repositories>
+    <repository>
+        <id>universityOfManchester-graal</id>
+        <url>https://raw.githubusercontent.com/beehive-lab/tornado/maven-tornadovm</url>
+    </repository>
+</repositories>
+
+<dependencies>
+<dependency>
+    <groupId>tornado</groupId>
+    <artifactId>tornado-api</artifactId>
+    <version>0.8</version>
+</dependency>
+<dependency>
+    <groupId>tornado</groupId>
+    <artifactId>tornado-matrices</artifactId>
+    <version>0.8</version>
+</dependency>
+</dependencies>
 ```
 
-To run TornadoVM, you need to either install the TornadoVM extension for GraalVM/OpenJDK, or run with our docker [images](assembly/src/docs/12_INSTALL_WITH_DOCKER.md).
+To run TornadoVM, you need to either install the TornadoVM extension for GraalVM/OpenJDK, or run with our
+docker [images](assembly/src/docs/12_INSTALL_WITH_DOCKER.md).
```

Comment:
run with our Docker image

Reply:
I don't get this. We have 6 docker images.

Reply:
sorry I meant to capitalize Docker